src/image-marker.js

```diff
@@ -0,0 +1,56 @@
+import { $ } from 'clappr'
+import StandardMarker from './standard-marker'
+
+/*
+ * An implementation of an image Marker, which can show an image.
+ */
+export default class ImageMarker extends StandardMarker {
+
+  /*
+   * time: the time in seconds that this marker represents
+   * tooltipImage: the image to be shown (optional)
+   * width: width for the image (Default: 200px)
+   * height: height for the image (Default: auto)
+   */
+  constructor (time, tooltipImage, width, height) {
+    super(time, tooltipImage)
+    this._tooltipText = tooltipImage || null
```

Comment:
Yes tried that but was never able to get it working since super() was invoking few methods in standardmarker which was not building markers correctly. Ended up copy pasting the whole class just to satisfy that one variable. see if you can refactor standard marker to move all the method calls to an init method instead of constructor.